addons/website/static/src/js/menu/edit.js

```diff
@@ -555,7 +555,7 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             {
                 groupName: _t('Website'),
                 title: _t('Separator'),
-                description: _t('Insert an horizontal separator sippet.'),
+                description: _t('Insert an horizontal separator snippet.'),
```

Comment:
Good catch :D
But pretty sure this one is a typo too
```suggestion
                description: _t('Insert a horizontal separator snippet.'),
```